ProcessMaker/Traits/HandlesValueAliasStatus.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace ProcessMaker\Traits;
+
+trait HandlesValueAliasStatus
+{
+    public function valueAliasStatus($value, $expression)
+    {
+        $statusMap = [
+            'in progress' => 'IN_PROGRESS',
```

Comment:
Use constants to handle this statuses. Create them if not exists.
```
ProcessRequest::REQUEST_STATUS_ACTIVE = 'ACTIVE';
CaseRepository::CASE_STATUS_ACTIVE = 'IN_PROGRESS';
```
Sync with @devmiguelangel to avoid conflict with PR https://github.com/ProcessMaker/processmaker/pull/7445